Exercise_A/index.html

```diff
@@ -41,5 +233,10 @@
       If you do not use CSS to do smooth scroll then... You need the CDN for Smooth Scroll
       Your custom JavaScript
      -->
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2"
+      crossorigin="anonymous"
+    ></script>
   </body>

```

Comment:
Great work with the website, but it seems like you forgot to add the part where it scrolls with the click of the button. You can wrap the button inside a <a href> and then link it to the bottom section using an ID